components/formats-bsd/src/loci/formats/in/DicomReader.java

```diff
@@ -65,78 +66,30 @@
 import ome.units.quantity.Length;
 import ome.units.UNITS;
 
+import static loci.formats.in.DicomAttribute.*;
+import static loci.formats.in.DicomVR.*;
```

Comment:
As mentioned during the weekly Formats discussion, immediate question is whether this API handling the specifics of the DICOM specification should be managed in a dedicated package similarly to `loci.formats.tiff`.

Reply:
625f8cc reorganizes into a new ```loci.formats.dicom``` package.